src/OVAL/probes/probe/main.c

```diff
@@ -271,16 +271,27 @@ int main(int argc, char *argv[])
 
 	pthread_attr_destroy(&th_attr);
 
+	probe_offline_mode();
+
 	/*
 	 * Setup offline mode(s)
 	 */
-	if ((rootdir = getenv("OSCAP_PROBE_ROOT")) != NULL) {
-		if(strlen(rootdir) > 0) {
+	rootdir = getenv("OSCAP_PROBE_ROOT");
+	if ((rootdir != NULL) && (strlen(rootdir) > 0)) {
+
+		preload_libraries_before_chroot(); // todo - maybe useless for own mode
```

Comment:
Probably. It will not break anything when we do it.

I would like to do cleanup after merging of this PR - use probe_offline_mode() in all probes, move `preload_libraries_before_chroot();` to some shared file.